src/units.cpp

```diff
@@ -426,15 +426,16 @@ void updateUnitsMap(const UnitsPtr &units, UnitsMap &unitsMap, double exp = 1.0)
             std::string pre;
             std::string id;
             double expMult;
-            units->unitAttributes(i, ref, pre, exp, expMult, id);
+            double e;
+            units->unitAttributes(i, ref, pre, e, expMult, id);
             if (isStandardUnitName(ref)) {
                 auto unit = standardUnitsList.find(ref);
```

Comment:
This is a poor choice of variable name.

Reply:
Yep, most of this is preliminary until I actually get code running (just OSX compiler mucking things up at the moment). I'll probably rename it to something like `uExp` for better clarity.